frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts

```diff
@@ -161,16 +161,14 @@ export class DataVolumeWrapper extends K8sResourceObjectWithTypePropertyWrapper<
   }
 
   protected sanitize(type: DataVolumeSourceType, { name, namespace, url }: CombinedTypeData) {
-    if (type === DataVolumeSourceType.BLANK) {
-      return {};
+    switch (type) {
+      case DataVolumeSourceType.HTTP:
+        return { url };
+      case DataVolumeSourceType.PVC:
+        return { name, namespace };
+      case DataVolumeSourceType.BLANK:
+      default:
+        return {};
```

Comment:
+1 - I definitely prefer the switch statements over multiple ifs in this situation.